README.md

```diff
@@ -1,46 +1,27 @@
-# java-gradle-starter-project
+# Weather station project
 
-Here is a starter project for the ArcGIS Maps SDK for Java with Gradle. 
+This project is a demonstration of en IoT weather station logger which integrates into the ArcGIS platform. It was featured in this blog post which gives more details of the developer technology we have used.
 
-The project includes the Gradle wrapper, so there is no need to install Gradle to run the app.
-
-The app launches a window displaying a map.
-
-![screenshot](screenshot.png)
-
-## Instructions
-
-### IntelliJ IDEA
+It has been designed to work on a Raspberry Pi Model 3B+ (or higher model) with a USB GPS receiver.
 
-1. Open IntelliJ IDEA and select _File > Open..._.
-2. Choose the java-gradle-starter-project directory and click _OK_.
-3. Select _File > Project Structure..._ and ensure that the Project SDK and language level are set to use Java 11.
-4. Open the Gradle view with _View > Tool Windows > Gradle_.
-5. In the Gradle view, double-click `copyNatives` under _Tasks > build_. This will unpack the native library dependencies to $USER_HOME/.arcgis.
-6. In the Gradle view, double-click `run` under _Tasks > application_ to run the app.
+The project uses an early adopter version of the ArcGIS Maps SDK for Java specifically compiled to work with Arm Linux platforms such as the Raspberry Pi or Jenson Orin units. The Java Maps SDK with support for Arm Linux can be requested by emailing ARM64LinuxNative@esri.com. This will install the libraries needed for this app in a Maven Local instance. This is a temporary way of getting the SDK whilst it is in Beta. Subsequent releases will be available in a public maven repository.
```

Comment:
```suggestion
The project uses an early adopter version of the [ArcGIS Maps SDK for Java ](https://community.esri.com/t5/arcgis-maps-sdks-native-blog/arcgis-maps-sdk-for-java-with-support-for-arm64/ba-p/1304914)specifically compiled to work with Arm Linux platforms such as the Raspberry Pi or Jenson Orin units. 

The Java Maps SDK with support for Arm Linux can be requested by emailing ARM64LinuxNative@esri.com. This will install the libraries needed for this app in a Maven Local instance. This is a temporary way of getting the SDK whilst it is in Beta. Subsequent releases will be available in a public maven repository.
```